pkg/apis/serving/v1alpha1/llm_inference_service_types.go

```diff
@@ -245,11 +247,22 @@ type InferencePoolSpec struct {
 type ParallelismSpec struct {
 	// Tensor parallelism size.
 	// +optional
-	Tensor *int64 `json:"tensor,omitempty"`
+	Tensor *int32 `json:"tensor,omitempty"`
 	// Pipeline parallelism size.
 	// +optional
-	Pipeline *int64 `json:"pipeline,omitempty"`
-	// TODO more to be added ...
+	Pipeline *int32 `json:"pipeline,omitempty"`
+	// Data parallelism size.
+	// +optional
+	Data *int32 `json:"data,omitempty"`
+	// DataLocal data local parallelism size.
```

Comment:
Can you explain more what's data local parallelism size?

Reply:
Data is the total number of DP ranks, Data local is the number of DP ranks per pod

Reply:
This doc was helpful for me to understand it a little better https://docs.google.com/document/d/1mSYsWQEbp4Oq50ghFWUun7OgagbuUJivAC8Ds-pu-xU/edit

Reply:
One question: for non MOE models, does it help us?

Reply:
I'm not sure if there are advantages for non MoE models, however, DP can be used without enabling expert parallelism